src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -0,0 +1,55 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class OccurrenceWordPoc {
+    /*
+    Entering the path
+    once if the input is received it shows processing
+    and executes if the file is present or not present
+     */
+    public static void main(String[] args) {
+        if (args.length == 2) {
```

Comment:
As per requirement, only txt and json files should be supported. Please add a check for that as well.

Reply:
Updated has suggested